src/modules/did.js

```diff
@@ -82,6 +85,18 @@ class DIDModule {
     return await this.signAndSend(this.createRemoveTx(didRemoval, signature), waitForFinalization, params);
   }
 
+  /**
+   * Creates an attestation claim on chain for a specific DID
+   * @param {string} attester - Attester's DID
+   * @param {object} attestation - Attestation object with priority and iri
+   * @param {Signature} signature - Signature from existing key
+   */
+  async attestClaim(attester, attestation, signature, waitForFinalization = true, params = {}) {
```

Comment:
claim and attestation mean the same thing. `attestClaim` = `attestAttest`. Maybe `setAttestedDocument` would be easier to understand.

Reply:
Other possibilities:
- `setAttestation`
- `setClaim`
- `setAttestsDocumentContents`

Reply:
yep fair enough, i'll go with `setClaim` to be like the substrate call. initial idea with this method naming was to differentiate as a api.tx.attests call